### ISSUE TITLE
Not possible?

### ISSUE THREAD

**Issue by Dany-LF** [csharp-leaf/Leaf.xNet#71]
https://github.com/csharp-leaf/Leaf.xNet/issues/70

No i use this for api i want to get a value in json

**Comment by grandsilence** [comment]
You can use `Substring` method extension

Example json
```json
{"name":"Alex"}
```
```csharp
string name = json.Substring("\"name\":", "\"");
```

Or [use Newton Json Deserializer](https://www.newtonsoft.com/json/help/html/DeserializeObject.htm) for more complex data

**Comment by Dany-LF** [comment]
Yeah resolved with #70  sorry for ur time...
